keps/sig-auth/20190607-certificates-api.md

```diff
@@ -132,6 +133,31 @@ A typical successful issuance proceeds as follows.
    the `.Status.Certificate` field.
 1. The requestor observes the update, and stores the certificate locally.
 
+### Signers
+CSRs have a `signerName` field which is used to specify which signer the CSR creator wants to sign the certificate.
+To support migration from v1beta1 to v1, this required field will be defaulted in v1beta1, but not in v1:
+ 1. If it's a kubelet client certificate, it is assigned " kubernetes.io/kubelet-client".
+ 2. Otherwise, it is assign "kubernetes.io/legacy-unknown".
+
+Kubernetes provides the following well-known signers:
+ 1. kubernetes.io/kube-apiserver-client - signs certificates that will be honored as client-certs by the kube-apiserver.
+    Never auto-approved by kube-controller-manager.
+ 2. kubernetes.io/kubelet-client - signs client certificates that will be honored as client-certs by the kube-apiserver.
```

Comment:
`kubelet-client` sounds like a client-cert that can be used against the kubelet's API... wondering if there's a clearer name for this

Reply:
> `kubelet-client` sounds like a client-cert that can be used against the kubelet's API... wondering if there's a clearer name for this

open to a suggestion

Reply:
how about `kube-apiserver-client-kubelet`? it's similar to the above signer since kubelet is just a special type of client with tighter restrictions

Reply:
> how about kube-apiserver-client-kubelet? it's similar to the above signer since kubelet is just a special type of client with tighter restrictions

that sounds good to me